2023/terkumafanyam/README.md

```diff
@@ -1,12 +1,220 @@
+
+
+### Checklist: Accessibility features documentation
+
+### Blog post on my knowledge of accessibility
+
+### [A Gentle Guide to Accessibility](https://terkumafanyam.hashnode.dev/a-gentle-introduction-to-accessibility)
+
+![Hands surrounding accessibility icon](./assets/istockphoto-a-protect.jpg)
+
+Accessibility, what does this even mean, anyway? Is it important? Do I have to bother myself with it? if this is you, I have great news for you. You are not alone. Allow me to guide you through the world of web accessibility.
+
+The scope of Accessibility is broad. In this post, our focus is on Web accessibility.
+
+#### What is Web Accessibility?
+
+Web Accessibility is the practice of designing websites that everyone can use. This is regardless of their abilities or disabilities.
+
+#### Why does Web Accessibility Matter?
+
+Have you ever wondered how or if people with disabilities interact with websites? Well, with the aid of assistive technologies, they can interact with the web. But this depends on how accessible the websites are. Web accessibility makes it easier for these technologies to help the impaired.
+
+Let's look at an accessibility issue that is peculiar to my case. I have a visual condition, dry eyes syndrome, to be exact. Certain color contrasts don't work for me, they cause my eyes to hurt after a short while. Darker themes are much more accommodating for my eyes. Due to my eye condition, using sites that do not provide a theme customization option is not ideal for me. It will be a nightmare if my workflow depends on such sites.
+
+#### How can I tell if a Website is Accessible or Not?
+
+A website is accessible if it meets the standards of WCAG or ATAG (for the producers of the web content). Does this mean I have to refer to these guidelines every time I make use of a website? No, is the short answer. Accessibility Checkers help us to check the accessibility of websites. These tools check if websites follow the WCAG or ATAG. After the check, a generated report records all the accessibility issues found. Some authoring tools like the Wagtail CMS include in-built accessibility checkers.
+
+![A screenshot of accessibility report on stackoverflow's site.](./assets/Screenshot_20231011_095116.png)
+
+The image above is an example of what an accessibility report looks like. WAVE is what I used to check the accessibility of Stackoverflow's home page here.
+
+#### Conclusion
+
+So far, we've looked at what web accessibility is and why it is important. We also looked at how we can test for web accessibility. Feel free to explore further on web accessibility. You can also look into other aspects of accessibility.
+
+Cheers for making it to the end of our journey. :)
+
+
+## Blog post on Technical writing
+
+## [A Beginner's Guide to React: Understanding its Power and Setting Up Your First App](https://terkumafanyam.hashnode.dev/a-beginners-guide-to-react-understanding-its-power-and-setting-up-your-first-app)
```

Comment:
This seems to me like solid technical writing, but just one example. It’d have been better IMHO if you wrote about technical writing, so you could share multiple examples.

Reply:
Thank you for the feedback. I will work on making a better post. :)